tests/test_package/test_main.py

```diff
@@ -15,3 +15,7 @@ def test_main(extension, tmp_path):
     path = str((tmp_path / "sim").with_suffix(extension))
     save_sim_to_path(path)
     main([path, "--test_only"])
+
+
+def test_parser():
```

Comment:
Would be good to have some documentation for how to invoke this, so as it stands now, one would need to do
```
tidy3d parsed_lsf_to_tidy3d_test.py -l example.lsf
```
?

1. I'm worried that conflicts with the previous `tidy3d simulation.json data.hdf5` command somewhat. how does it know that the `parsed_lsf_to_tidy3d_test.py` is the output file name and not the input file name for the old main function?
2. I think it might be cleaner to introduce a new command like `tidy3d convert in_file out_file` to separate functionality (same comment as 1 but a suggestion for how to resolve it)
3. It seems a bit strange from my perspective to provide the output file first and the input file second.
4. Would be good to document somewhere (eg in the docs) how to use this, maybe in our [howdoi](https://docs.flexcompute.com/projects/tidy3d/en/latest/howdoi.html) section of the docs?

Reply:
Sounds good. I think I initially designed it this way because it looked like a "sim" argument was required in __main__.py, so I figured I would use that, but there is probably a way to implement it in the way you suggested in 2.

Reply:
I think so, we have a `tidy3d configure` command defined in `tidy3d/web/cli` that has its own separate functionality (setting the user API key) so we should be able to just follow that example with `tidy3d convert` and then there should be no overlap with just the `tidy3d ...` command I think (which runs simulation.json files directly, not scripts)